src/components/common/ProductBox/ProductBox.js

```diff
@@ -47,10 +50,26 @@ const ProductBox = ({
     <div className={styles.line}></div>
     <div className={styles.actions}>
       <div className={styles.outlines}>
-        <Button className={favourite ? styles.active : ''} variant='outline'>
+        <Button
+          onClick={
+            favourite === true
+              ? () => setFavouriteValue(id, false)
```

Comment:
Strona po dodaniu skacze do góry i zmienia się URL, brakuje `preventDefault`

Reply:
Nie użyłem tutaj preventDefault ponieważ wtedy  nie zadziałałyby akcje reduxowe. Postawiłem więc na alternatywny sposób, aby jako adres linku użyć javascript: void(0), dało to oczekiwany efekt.